L1Trigger/L1CaloTrigger/test/Phase1L1TJetHwEmuComp.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import sys
+import math
+import numpy
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import colors
+
+ptLSB = 0.25;
+etaLSB = 0.0043633231;
+phiLSB = 0.0043633231;
+hwJets = []
+emJets = []
+hwData = []
+emData = []
+hwDataNoZ = []
+emDataNoZ = []
+nHw = 0
+nEm = 0
+nEv = 5000
+nEvNoZ = 0
+
+
+for hwRx in xrange(0,70):
```

Comment:
xrange does not exist in python3. Just use range.

Reply:
done